ui/src/app/tasks/task-schedule-create/task-schedule-create.validator.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { FormArray, FormControl } from '@angular/forms';
+import { TaskScheduleCreateValidator } from './task-schedule-create.validator';
+
+/**
+ * Test Task Schedule Create Validator functions {TaskScheduleCreateValidator}.
+ *
+ * @author Damien Vitrac
+ */
+describe('TaskScheduleCreateValidator', () => {
+
+  describe('cron', () => {
+
+    it('invalid', () => {
+      [].forEach((mock) => {
+        const form: FormControl = new FormControl(mock);
+        expect(TaskScheduleCreateValidator.cron(form)).toBeTruthy();
+      });
+    });
+
+    it('valid', () => {
+      [
+        null,
+        '0 0 0 0 0 0 0',
+        '0 15 10 * * ? 2005',
+        '0 15 10 ? * 6L 2002-2005',
+        '0 15 10 ? * MON-FRI',
+        '0 15 10 ? * 6#3',
+        '0 11 11 11 11 ?',
+        '0 0 12 * * ?',
```

Comment:
According the PCF docs, cron expressions with more than 5 fields should be disallowed. Currently it even throws a nasty exception. See: https://docs.pivotal.io/pcf-scheduler/1-2/using-jobs.html#create-jobs under `Schedule a Job`

Reply:
@ghillert: We aren't going to be validating anything on the client side. We can, however, point to an external link (in cf docs) for someone wanting to learn more about the syntax correctness.